peps/pep-0694.rst

```diff
@@ -903,6 +925,11 @@ Required File Upload Mechanisms
 
 Upload API version 2.0 compliant servers **MUST** support the ``http-post-bytes`` mechanism.
 
+This mechanism **MUST** use the same authentication scheme as the
```

Comment:
Would it be better to say something like "All mechanisms **MUST** use the same authentication scheme as the rest of the Upload 2.0 protocol" or is that too restrictive?  I'm thinking about the pre-signed S3 bucket case where the index wraps the overall protocol, but the actual mechanics of the file upload is handled by a different service.

Otherwise, what you have here sounds good to me.

Reply:
I would prefer to leave authentication for non-pep specified mechanisms up to the implementers. There may also be a case where future required mechanisms may also have alternative authentication mechanisms, so it likely is too restrictive in my opinion.